packages/admin/admin-theme/src/componentsTheme/MuiButton.ts

```diff
@@ -65,6 +73,11 @@ export const getMuiButton: GetMuiComponentTheme<"MuiButton"> = (component, { pal
                 backgroundColor: palette.primary.dark,
                 borderColor: palette.primary.dark,
             },
+
+            "&:focus": {
```

Comment:
Please remove MUIs default focus-circle thing as it's not part of our design. 

| Current implementation | Design |
| --- | --- | 
| <img width="138" alt="Focus-Button implementation" src="https://github.com/user-attachments/assets/45317f4a-7f04-4669-9397-d28e402aa556" /> | <img width="132" alt="Focus-Button design" src="https://github.com/user-attachments/assets/0f42584c-ddd0-4a58-bf1e-b2a981796b84" /> |